### ISSUE TITLE
Feature Request: Primary intelligence requirements (PIR) tracking

### ISSUE THREAD

**Issue by syloktools** [MISP/MISP#8658]
### Is your feature request related to a problem? Please describe.

I have yet to find a threat intelligence tool that tracks or takes into account your PIRs. It would be good to be able to set your PIRs in your main MISP instance and track events that meet the collection requirements of a certain PIR. 

### Describe the solution you'd like

This would require a separate view where you list your PIRs and the tags, galaxies, and keywords and even source (feed/org) that would be considered a match. When an event in ingested, a process looks at the event and give it a score as to how closely it matches PIR 1, PIR2, etc. Match percentage would be based on number of matching elements. You could give more "weight" to certain matches than others. There would also be a view where you can see your PIRs vs collection sources/orgs. 

Basically a collection management framework section inside MISP. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
